src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/DebugPropertyPage.xaml.cs.xlf

```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="Category|General|DisplayName">
         <source>General</source>
-        <target state="new">Obecné</target>
+        <target state="new">General</target>

```

Comment:
This looks strange. The translation is undone, while the resource is marked as triaged.

I remember seeing a bug related to "General" appearing in the UI, untranslated.

@tmeschter @melytc @MiYanni any insight here?

Reply:
The same changes appear in PR #8020 too. The PRs appear to be duplicates.

Reply:
This seems to be related to #8017.